.ci/pytorch/common_utils.sh

```diff
@@ -283,6 +283,11 @@ function clone_pytorch_xla() {
   fi
 }
 
+function install_transformers() {
+  pip install transformers==4.54.0
```

Comment:
Just a note, the latest is 4.55 now that supports the fresh-off-the-press `gpt-oss`